circuits/src/mpc_circuits/match.rs

```diff
@@ -23,12 +23,20 @@ use crate::mpc_gadgets::{comparators::min, fixed_point::FixedPointMpcGadget};
 /// So, if the match result is invalid, the orders don't overlap, etc; the result will
 /// never be opened, and the information never leaked. Therefore, we do not need to zero
 /// out any values in the circuit.
+///
+/// The fee terms are calculated outside of the circuit and passed in directly by the relayers.
+/// For a relayer that intends to sell the base asset, this term is directly the fee authorized by
+/// the user. For a relayer that intends to sell the quote asset, a unit change is implicitly applied
```

Comment:
nit: it may aid readability to phrase this is the unit change being "assumed to be applied" as opposed to "implicitly applied", and mention that this assumption is constrained in the match proof

(this would also be consistent with the great doc comment below)